workspaces/src/worker/impls.rs

```diff
@@ -174,16 +177,58 @@ impl Worker<Sandbox> {
         self.workspace.import_contract(id, worker)
     }
 
+    /// Patch state into the sandbox network, using builder pattern. This will allow us to set
+    /// state that we have acquired in some manner. This allows us to test random cases that
+    /// are hard to come up naturally as state evolves.
+    pub fn patch_state_builder(&self, account_id: &AccountId) -> SandboxPatchStateBuilder {
+        self.workspace.patch_state(account_id.clone())
+    }
+
+    /// Patch account state using builder pattern
+    pub fn patch_account(&self, account_id: &AccountId) -> SandboxPatchStateAccountBuilder {
+        self.workspace.patch_account(account_id.clone())
+    }
+
     /// Patch state into the sandbox network, given a key and value. This will allow us to set
     /// state that we have acquired in some manner. This allows us to test random cases that
     /// are hard to come up naturally as state evolves.
     pub async fn patch_state(
         &self,
         contract_id: &AccountId,
-        key: &[u8],
-        value: &[u8],
+        key: impl Into<Vec<u8>>,
+        value: impl Into<Vec<u8>>,
```

Comment:
not everything will impl an `Into<Vec<u8>>` and by that notion won’t fit all cases. Best to keep as-is to avoid that situation, since in rust a lot of things can be converted into a series of bytes `&[u8]` including `Vec<u8>`. This will also avoid having to make a breaking change for this since now people would have to require this trait impl.

Reply:
Yeah these traits are not as intuitive for a developer to read as well as it sticks us with using this API even when we remove the requirement for bytes to be converted to `Vec<u8>`